package.json

```diff
@@ -15,28 +15,34 @@
     "start": "next start"
   },
   "dependencies": {
-    "@auth/prisma-adapter": "^1.6.0",
```

Comment:
See this here? This was 0.0 when we looked at it the other night. At some point it must have broken before we updated it